src/js/jsx/search/SearchBar.jsx

```diff
@@ -216,7 +215,8 @@ define(function (require, exports, module) {
          */
         _updateDatalistInput: function (id) {
             if (id) {
-                var currFilter = this.refs.datalist.getInputValue().split(" "),
+                var partialMatch,
```

Comment:
Style nit: in var declaration blocks, put the declarations with definitions first.